requirements/base.in

```diff
@@ -0,0 +1,11 @@
+dj-database-url~=0.5
+gunicorn~=19.9
+psycopg2-binary~=2.8
+pytz==2020.1
+whitenoise~=4.1
+django-crispy-forms~=1.8
+Django~=2.2
+django-registration~=3.0
+django-sendgrid-v5~=0.8
+djangorestframework~=3.10
+django-js-reverse~=0.9
```

Comment:
I'm going to take on faith that these requirements are correct, as Github provides no straightforward way to review this kind of change.

Reply:
They're the same as it was, just changed to `~=`s.